### PR TITLE
Update datadog docs

### PR DESCRIPTION
**Pull request by zalbiraw**
## Related Issue (if applicable)

Update datadog docs about logs and analytics

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

**Review by JRWu — APPROVED** [review]
LGTM, short and sweet

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+date: 2017-03-27T16:30:52+01:00
+title: Datadog logs show up under the error status
+menu:
+  main:
+    parent: "Frequently Asked Questions"
+weight: 0 
+---
+
+To allow Datadog to read the Tyk logs correctly. We need to create a pipeline to process the data and allow Datadog to understand that data.
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]: Please add something to clarify that you need a running tyk deployment that already sends data to datadog (with Datadog Agent)

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+To do that, we need to access the `/logs/pipelines` path on your datadog web application. This will take us to the pipeline configuration page. In here, we will create a new pipeline. For the filter section, use `Service:tyk-*` this will capture logs for any of the Tyk related services.
+
+{{< img src="/img/faq/datadog-logs-showup-as-errors/create-pipeline.png" alt="Create Datadog Logs Pipeline to process Tyk services' logs" >}}
+
+Next we will need to add processor to the pipeline. Select the Grok Parser processor type, give it a name and click on the `Parse My Logs` button and `Create` .
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]: It is not clear that you need to expand the pipeline that you just created and there you can click to create the parser 

image

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+date: 2017-03-27T16:30:52+01:00
+title: Datadog logs show up under the error status
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]: I don't fully understand the title. 
Also please add a description

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+menu:
+  main:
+    parent: "Frequently Asked Questions"
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: Any reason for not putting this under `troubleshooting/tyk-pump/` section? or even better - move it to a section in `tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration/datadog.md`

**Review by letzya — CHANGES_REQUESTED** [review]
I added some content to tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration/datadog.md

Please review my other comments

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2017-03-27T16:30:52+01:00
+title: Datadog logs show up under the error status
```
Comment (zalbiraw) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]:
```suggestion
title: Tyk logs scraped with Datadog agent show up as errors under Datadog logs 
```

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md (hunk 4 above)
Comment (zalbiraw) [reply tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: This is not part of the pump configuration. This has to do with datadog agent scraping logs and those logs not being parsed properly by dd

Comment (letzya) [reply tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: ok, so please put it under troubleshooting/tyk-pump/

Comment (zalbiraw) [reply tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: I think you misunderstood this is not relevant to pump. It scraps all container logs.

Comment (letzya) [reply tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: ok gotcha.I think we need to make this clearer. 
his FAQ's title is about getting errors `Tyk logs scraped with Datadog agent show up as errors under Datadog` - maybe it should be `Send Tyk container logs to Datadogs `

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2017-03-27T16:30:52+01:00
+title: Tyk logs scraped with Datadog agent show up as errors under Datadog logs 
+menu:
+  main:
+    parent: "Frequently Asked Questions"
+weight: 0 
+---
+
+To allow Datadog Agent to scrape the logs of your Tyk deployment correctly.
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]:
```suggestion
With Datadog you can view logs of all your Tyk components.
To allow Datadog Agent to scrape the logs of your Tyk deployment correctly you need to create a pipeline in Datadog, to process and underst and this data.
```

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+We need to create a pipeline to process the data and allow Datadog to understand that data.
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]:
```suggestion
```

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md (hunk 4 above)
Comment (zalbiraw) [reply tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: The question that the user would be asking is `why are tyk logs showing up as errors in my datadog logs` the datadog agent integration is going to pick up the tyk container logs by default and those by default will show up as errors. So maybe we can do `all tyk container logs show up under the error status in Datadog logs`

Comment (letzya) [reply tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md:6]: ok, now i remember, I thought I disabled the collection of the logs in my configuration doc, but maybe it was just something i did in my test env

File: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2017-03-27T16:30:52+01:00
+title: Tyk logs scraped with Datadog agent show up as errors under Datadog logs
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md]:
```suggestion
title: All tyk container logs show up under the error status in Datadog logs
```

**Review by letzya — APPROVED** [review]

**Pull request merged**